api-get-options.go

```diff
@@ -19,81 +19,98 @@ package minio
 import (
 	"fmt"
 	"net/http"
+	"strings"
 	"time"
+
+	"github.com/minio/minio-go/pkg/encrypt"
 )
 
-// RequestHeaders - implement methods for setting special
-// request headers for GET, HEAD object operations.
-// http://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectGET.html
-type RequestHeaders struct {
-	http.Header
+// GetOptions are used to specify additional headers or options
+// during GET requests.
+type GetOptions struct {
```

Comment:
Can we rename this GetObjectOptions to be symmetric to the PutObjectOptions

Reply:
My plan was to rename `PutObjectOptions` -> `PutOptions` during encryption refactoring.
I can ask Harsha

Reply:
Actually it is more consistent to keep `GetObjectOptions()` instead of `GetOptions()` GetOptions is just too generic.

Reply:
Okay